src/domain/service/user.services.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, Logger } from "@nestjs/common";
+import { CreateUserDto } from "src/application/dto/users/create_user.dto";
+import { UserRepository } from "src/infrastructure/repository/user.repository";
+
+@Injectable()
+export class UserService {
+  private logger = new Logger('UserService', { timestamp: true });
+
+  constructor(
+    private readonly userRepository: UserRepository,
+  ) {}
+
+  async createUser(body: CreateUserDto){
+    this.userRepository.createUser(body)
+  }
+
+  async getUsers(){
+    return this.userRepository.getUsers();
```

Comment:
I think we need use DTO to map response. Please not return entity

Reply:
I will create another branch to config this